ecs-cli/modules/config/readwriter.go

```diff
@@ -68,14 +59,53 @@ func NewReadWriter() (*IniReadWriter, error) {
 }
 
 // GetConfig gets the ecs-cli config object from the config file.
-func (rdwr *IniReadWriter) GetConfig() (*CliConfig, error) {
-	to := new(CliConfig)
-	err := rdwr.cfg.MapTo(to)
+func (rdwr *IniReadWriter) GetConfig() (*CliConfig, map[interface{}]interface{}, error) {
```

Comment:
Why is the second return type `map[interface{}]interface{}`? Can it contain more specific types? Can you add some documentation detailing what's being returned from this method?